app/models/pydantic/creation_options.py

```diff
@@ -190,6 +190,13 @@ class VectorSourceCreationOptions(StrictBaseModel):
         ],
         description="List of indices to add to table",
     )
+    cluster: Optional[Index] = Field(
+        None, description="Index to use for clustering (optional)."
```

Comment:
```suggestion
        None, description="Index to use for clustering."
```

Reply:
The Optional data type should already indicate that it is optional. In the documentation required attributes will be automatically highlighted with a red asterisk, so no need to mention this here again.